hatchery/hatchery.go

```diff
@@ -26,6 +27,7 @@ func RegisterHatchery(mux *httptrace.ServeMux) {
 	mux.HandleFunc("/options", options)
 	mux.HandleFunc("/paymodels", paymodels)
 	mux.HandleFunc("/setpaymodel", setpaymodel)
+	mux.HandleFunc("/resetpaymodels", resetPaymodels)
```

Comment:
We don't call them from outside at the moment. So we don't need to, if we are not supposed to.

Reply:
If we have to take it down for some reason, is it enough if we just remove the above line, instead of the whole function implementation? That saves us some time if we ever have to have it in the future. 

🫣ahem!  I can hear myself speaking completely against the YAGNI principle, so lmk even if you want the whole endpoint and its unit test down.

Reply:
We can leave the endpoint in for now. Maybe we can have a reset paymodel button in the portal for the future.